man/fi_shm.7.md

```diff
@@ -163,12 +163,6 @@ The *shm* provider checks for the following environment variables:
 *FI_SHM_USE_DSA_SAR*
 : Enables memory copy offload to Intel DSA in SAR protocol. Default false
 
-*FI_SHM_ENABLE_DSA_PAGE_TOUCH*
-: Enables CPU touching of memory pages in a DSA command descriptor when the
-  page fault is reported, so that there is valid address translation for the
-  remaining addresses in the command. This minimizes DSA page faults. Default
-  false
-
 *FI_SHM_USE_XPMEM*
```

Comment:
Please explain in the commit message why the changes are made.

Reply:
Got it. Explained this, and also updated the code to use DSA_COMP_NONE. Pushed updates.